### ISSUE TITLE
Where are the config files?

### ISSUE THREAD

**Issue by i-am-linja** [alexbatalov/fallout1-ce#205]
The README mentions `fallout.cfg` and `f1_res.ini`, but nothing I've seen anywhere tells me _where_ I should save those files for any effect.

**Comment by xparaly** [comment]
I'm on PC and I got Fallout from Steam.  fallout.cfg and f1_res.ini were both already in my fallout directory.  I believe you just need the files to be in the same folder as fallout1-ce.

![image](https://github.com/user-attachments/assets/349c8a91-3fc8-416c-ac86-9db22f7c0a51)

**Comment by i-am-linja** [comment]
Ok, that is helpful. Do you mind posting examples of those files as well? In particular, do I need a [MAIN] header in fallout.cfg?

**Comment by i-am-linja** [comment]
I'm on macOS, using the GOG distribution, and I just found mine in `__support/app/`. Will the app read them directly from there or do I need to move them?

**Comment by i-am-linja** [comment]
Upon closer inspection, the extracted contents of both games are kind of all over the place. Fallout I has everything immediately at the top level, with near-stub `app`, `commonappdata`, and `tmp` folders; Fallout II has only those three folders at the top level, with everything at I's top level in `app`. I have no idea what's where.

**Comment by xparaly** [comment]
> I'm on macOS, using the GOG distribution, and I just found mine in `__support/app/`. Will the app read them directly from there or do I need to move them?

I don't have any reason to believe you need to move the config files anywhere.  Why are you concerned about where the config files are in the first place?  Have you tried running fallout-ce?

**Comment by i-am-linja** [comment]
I did in fact need to move the config files, into the top level, for them to have any effect. And in fact when I ran the app without the files, the display occupied about one-sixteenth of my screen. Same thing happened with the files, regard less of scaling setting.

I couldn’t get the display satisfactory, so ultimately I returned the base game. There is now enough information in this thread to help anyone with the original issue, so closing. I’m going to refrain from buying any more games for a while.
